app/src/test/java/cmm/apps/esmorga/di/DIModulesTest.kt

```diff
@@ -23,10 +26,14 @@ class DIModulesTest {
     @Test
     fun verifyKoinApp() {
         koinApplication {
+            MockProvider.register {
```

Comment:
I'm not a mock expert xD, so what are you trying to do here? Since it seems to a modules test class, not for data class models